discovery/roles/discovery_validations/common/tasks/include_high_availability_config.yml

```diff
@@ -50,7 +50,12 @@
   ansible.builtin.set_fact:
     nfs_used: "{{ omnia_share_option == 'NFS' }}"
 
-- name: Fail if enable OIM is true, but NFS is not configured
+- name: Fail if enable_OIM_ha is true, but NFS is not configured
+  when: enable_oim_ha and not nfs_used
```

Comment:
why 2 failure require? use single when condition for oim ha, service ha and hierarchical cluster